EpiserverRedirects/Repository/DynamicDataStoreRepository.cs

```diff
@@ -46,6 +72,24 @@ public override RedirectRule Update(RedirectRule redirectRule)
             return redirectRuleToUpdate;
         }
 
+        public override bool RemoveAllDuplicates()
+        {
+            var redirectRules = GetAll();
+        
+            try
+            {
+                foreach (var redirectRule in redirectRules)
+                {
+                    RemoveDuplicates(redirectRule);
+                }
+                return true;
+            }
+            catch
+            {
+                return false;
```

Comment:
I feel that at least what we should do here is to log error through ILogger - I know errors are ignored in every other method (ie Delete, ClearAll) but maybe we should fix that? What do you think @m-jedynak 
(Even better solution is to return errror message to end user)